internal/fuzzbinder/fuzzgenerator.go

```diff
@@ -125,7 +125,7 @@ func GenerateFuzzClaim(fuzzParameters *FuzzParameters) (*intoto.Statement, error
 	currentTime := time.Now()
 	tomorrow := time.Now().AddDate(0, 0, 1)
 	// TODO(#173): Add validity duration as an input parameter.
-	endValidity := time.Now().AddDate(0, 0, 7)
+	endValidity := time.Now().AddDate(0, 0, validityDays)
```

Comment:
Can validityDays be any number? Or is there a range limit?

Reply:
I meant if AddDate makes any assumptions about the number of days. I hope it does not. For instance, adding 40 days is the same as 1 month and 9-10 days.

Reply:
No, there is no assumption about the number of days from what I have seen.